packages/dev/core/src/Materials/material.ts

```diff
@@ -1147,13 +1147,22 @@ export class Material implements IAnimatable, IClipPlanesHolder {
         return false;
     }
 
+    /**
+     * Specifies whether or not this material should be rendered in alpha test mode when used with the given mesh.
+     * @param mesh defines the mesh to check
+     * @returns a boolean specifying if an alpha test is needed.
+     */
+    public needAlphaTestingForMesh(mesh: AbstractMesh): boolean {

```

Comment:
Why public? Are you expecting it to be used elsewhere?

Reply:
Just following the pattern. `needAlphaBlendingForMesh` is also public. If we do this, we probably should be calling this function from other places that are currently calling `needAlphaTesting`.

Reply:
Yes, all current calls to `needAlphaTesting` (~10 after quick inspection) should probably be replaced by calls to `needAlphaTestingForMesh`.

Reply:
I've updated the code to use the new function. Still draft until the sample model is merged so that I can add a test, but the code part should be good now.